src/model/remoteConfig.ts

```diff
@@ -226,10 +226,11 @@ export async function fetchRemoteConfig(): Promise<RainbowConfig> {
     });
     return config;
   } catch (e) {
+    if (e instanceof Error && e.message === 'The request timed out') return DEFAULT_CONFIG;
```

Comment:
What's the idea behind not returning the most recent remote config for this type of error specifically?

Reply:
nope good catch